x-pack/plugins/aiops/public/components/change_point_detection/split_field_selector.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, useMemo } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiFormRow, EuiSelect, type EuiSelectOption } from '@elastic/eui';
+import { useChangePontDetectionContext } from './change_point_detection_context';
+
+interface SplitFieldSelectorProps {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+export const SplitFieldSelector: FC<SplitFieldSelectorProps> = React.memo(({ value, onChange }) => {
+  const { splitFieldsOptions } = useChangePontDetectionContext();
+
+  const options = useMemo<EuiSelectOption[]>(() => {
+    return splitFieldsOptions.map((v) => ({ value: v.name, text: v.displayName }));
+  }, [splitFieldsOptions]);
+
+  return (
+    <EuiFormRow
+      label={
+        <FormattedMessage
+          id="xpack.aiops.changePointDetection.selectSpitFieldLabel"
+          defaultMessage="Split field"
+        />
+      }
+    >
+      <EuiSelect options={options} value={value} onChange={(e) => onChange(e.target.value)} />
```

Comment:
Suggestion maybe for a follow up: For datasets with a lot of attributes, it might be useful to use a selector that allows searching the fields.

Reply:
It's a good point, I'm going to replace select with a [combo-box](https://elastic.github.io/eui/#/forms/combo-box)